### ISSUE TITLE
Link WCAG2 doesn't work

### ISSUE THREAD

**Issue by smartm0use** [validator/validator#454]
Hello, i'm here to report that link to WCAG2 test doesn't work.

The link that doesn't work is the WCAG checker from "**Check for WCAG 2.0 parsing compliance: filters checker results to just show messages relevant to requirements in the WCAG 2.0 spec**" from the following page: https://validator.w3.org/nu/about.html.

Can you fix it?

**Comment by sideshowbarker** [comment]
The **Check for WCAG 2.0 parsing compliance** link there a [bookmarklet](https://en.wikipedia.org/wiki/Bookmarklet) that you’re meant to [drag and drop to your browser’s bookmark bar](https://en.wikipedia.org/wiki/Bookmarklet#Installation).

**Comment by smartm0use** [comment]
Even adding it to my browser's bookmark bar it doesn't work. What should I see as output?
Could it be better to create a script that uses `achecker.ca` site as checker place where to POST the source page?
I mean something similar to W3C bookmarklet...

**Comment by sideshowbarker** [comment]
See https://www.paciellogroup.com/blog/2012/02/wcag-2-0-parsing-error-bookmarklet/

The way this bookmarklet needs to be used is after you check a document with the checker and it shows the results page. Then if you press the button for the bookmarklet in your bookmarks bar, it will (should) filter the results to show just the errors/warnings that are WCAG-related.
